tests/median_calculation/random_vector_median_calculation.cpp

```diff
@@ -145,8 +155,8 @@ int main(int argc, char **argv) {
       double x_intercept = x_start_dist(rnd_engine);
       double y_intercept = y_start_dist(rnd_engine);
 
-      double x_slope = x_beta_dist(rnd_engine) * plus_or_minus(rnd_engine) * 25;
-      double y_slope = y_beta_dist(rnd_engine) * plus_or_minus(rnd_engine) * 25;
+      double x_slope = x_beta_dist(rnd_engine) * plus_or_minus(rnd_engine) * 2;
```

Comment:
Please put comment about what the constant multiplier means.

Reply:
@KeitaRckPn, would you add a comment to this and just commit/push to this branch?

Thanks,